src/util/date-input-helper.ts

```diff
@@ -42,6 +44,14 @@ export namespace DateInputHelper {
             return false;
         }
 
+        if (
```

Comment:
combine this with the previous block since there's overlapping conditions
```
if (between && between.length) {
  if invalid format for between
        return false
  if start or end are invalid
       return false
}
```